app/views/posts/index.html.erb

```diff
@@ -1,29 +1,23 @@
 <header>
-  <h1>Username</h1>
+  <h1><%= @user.name %></h1>
   <hr/>
 </header>
 
 <section>
-  <h2>Recent Posts</h2>
-  <div>
-    <h3>Post 1</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
-    <div>
-      <p>Username: Comments 1</p>
-      <p>Username: Comments 2</p>
-      <p>Username: Comments 3</p>
-    </div>
-  </div>
-  <div>
-    <h3>Post 2</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
-    <div>
-    <p>Username: Comments 1</p>
-    <p>Username: Comments 2</p>
-    <p>Username: Comments 3</p>
-  </div>
-  </div>
-  <button>Pagination</button>
+  <% @user.posts.each do |post| %>
+    <article>
+      <h2>
+      <%= link_to link_to post.title, user_post_path(@user.id, post.id) %>
+      </h2>
+      <p><%= post.text %></p>
+      <span>Comments: <%= post.comments_counter%>,</span>
+      <span>Likes: <%= post.likes_counter %></span>
+      <% post.comments.each do |comment| %>
+        <p>Username: <%= comment.text %></p>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use `five_recent_comments` method to render the 5 most recent comments.

Reply:
Alright, I will make the changes